src/battle_util.c

```diff
@@ -1567,8 +1569,10 @@ void PrepareStringBattle(u16 stringId, u8 battler)
               && ((targetAbility == ABILITY_DEFIANT && CompareStat(gBattlerTarget, STAT_ATK, MAX_STAT_STAGE, CMP_LESS_THAN))
                  || (targetAbility == ABILITY_COMPETITIVE && CompareStat(gBattlerTarget, STAT_SPATK, MAX_STAT_STAGE, CMP_LESS_THAN)))
               && gSpecialStatuses[gBattlerTarget].changedStatsBattlerId != BATTLE_PARTNER(gBattlerTarget)
-              && gSpecialStatuses[gBattlerTarget].changedStatsBattlerId != gBattlerTarget)
+              // && gSpecialStatuses[gBattlerTarget].changedStatsBattlerId != gBattlerTarget) // Why was this here? Needs investigating
```

Comment:
isn't this here to prevent defiant from activating when a battler lowers its own stat?

Reply:
Ah yeah seems it needs to be there for curse (but not superpower etc.). It's also why sticky web doesn't trigger defiant, will see what I can do about it.